RecoHI/HiJetAlgos/src/MultipleAlgoIterator.cc

```diff
@@ -3,10 +3,19 @@
 #include "RecoHI/HiJetAlgos/interface/MultipleAlgoIterator.h"
 #include "DataFormats/Candidate/interface/CandidateFwd.h"
 #include "DataFormats/Candidate/interface/Candidate.h"
+#include "DataFormats/Math/interface/deltaR.h"
 
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 using namespace std;
 
+MultipleAlgoIterator::MultipleAlgoIterator(const edm::ParameterSet& iConfig, edm::ConsumesCollector&& iC)
+    : PileUpSubtractor(iConfig, std::move(iC)),
+      minimumTowersFraction_(iConfig.getParameter<double>("minimumTowersFraction")),
+      sumRecHits_(iConfig.getParameter<bool>("sumRecHits")),
+      dropZeroTowers_(iConfig.getUntrackedParameter<bool>("dropZeroTowers", true)) {
```

Comment:
it looks like this changes the results, and as such it should be made tracked.
I see that this is inherited from the past and will affect some other code in this package.
So, if really not convenient in this PR, please plan to have a follow up.